hoomd/mpcd/stream.py

```diff
@@ -693,6 +693,115 @@ def set_params(self, R=None, boundary=None):
             self.boundary = boundary
 
         bc = self._process_boundary(self.boundary)
-        self._cpp.geometry = _mpcd.SphereGeometry(self.R,bc)
+        self._cpp.geometry = _mpcd.SphereGeometry(self.R,0.0,bc)
         if self._filler is not None:
             self._filler.setGeometry(self._cpp.geometry)
+
+class dryingsphere(_streaming_method):
+    r""" Drying Spherical streaming geometry.
+
+    Args:
+        R (variant): confinement radius
+        density (float): solvent number density inside sphere
+        boundary (str): boundary condition at wall ("slip" or "no_slip")
+        period (int): Number of integration steps between collisions
+
+    The dryingsphere class models a fluid confined inside a drying sphere, centered at the
+    origin and sphere size is decreasing according to radius variant R. Solvent particles are
+    reflected from the spherical walls using appropriate boundary conditions.
+
+    Examples::
+        stream.dryingsphere(period=10, R= hoomd.variant(variant for R), density =5.,seed=394 )
+
+
+    Note: You can't change the Radius and density and boundary once you have initialised it.
+    
+    """
+    def __init__(self, R, density , boundary="no_slip", period=1, seed = 234):
+        hoomd.util.print_status_line()
+
+        _streaming_method.__init__(self, period)
+
+        self.metadata_fields += ['R','density','boundary','seed']
+        self.R = hoomd.variant._setup_variant_input(R)
+        self.density = density
+        self.boundary = boundary
+        self.seed = seed
+
+        bc = self._process_boundary(boundary)
+
+        # create the base streaming class
+        if not hoomd.context.exec_conf.isCUDAEnabled():
+            stream_class = _mpcd.DryingDropletStreamingMethod
+        else:
+            stream_class = _mpcd.ConfinedStreamingMethodGPUSphere
+        self._cpp = stream_class(hoomd.context.current.mpcd.data,
+                                 hoomd.context.current.system.getCurrentTimeStep(),
+                                 self.period,
+                                 0,
+                                 self.R.cpp_variant,
+                                 self.density,
+                                 self.seed,
+                                 bc)
+
+    def set_filler(self, density, kT, seed, type='A'):
```

Comment:
This is going to cause issues because we need to sync up the geometries, will come back to this later.

Reply:
Okay Thanks , so currently I am removing set_filler and remove_filler from here